plugin/src/arch/aarch64/aarch64data.rs

```diff
@@ -83,11 +83,11 @@ pub enum Matcher {
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub enum Command {
     // commands that advance the argument pointer
-    R(u8), // encode a register, or reference base, into a 5-bit bitfield.
+    R(u8),     // encode a register, or reference base, into a 5-bit bitfield.
```

Comment:
This is what I mean with rustfmt disliking vertical alignment. The comments here were indented to the same line for readability.